mingw-moz/setup

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -x # Show commands
+set -eu # Errors/undefined vars are fatal
+set -o pipefail # Check all commands in a pipeline
+
+date
+
+echo Downloading mingw repo
+pushd $INDEX_ROOT
+if [ -d "git" ]
+then
+    echo "Found pre-existing folder; skipping re-download."
+else
+    wget -nv https://s3-us-west-2.amazonaws.com/searchfox.repositories/mingw.tar
+    tar xf mingw.tar
+    rm mingw.tar
+fi
+popd
+
+date
+
+echo Downloading mingw blame.
+pushd $INDEX_ROOT
+if [ -d "blame" ]
+then
+    echo "Found pre-existing blame folder; skipping re-download."
+else
+    wget -nv https://s3-us-west-2.amazonaws.com/searchfox.repositories/mingw-blame.tar
+    tar xf mingw-blame.tar
+    rm mingw-blame.tar
+fi
+popd
+
+date
+
+MOZ_REV=$(curl -SsfL "https://hg.mozilla.org/mozilla-central/raw-file/tip/taskcluster/ci/fetch/toolchains.yml" | yq read - "mingw-w64.fetch.revision")
```

Comment:
I'm assuming there will be a separate pull request that adds `yq` to be installed in `indexer-provision.sh` (and the indexers re-provisioned) or in `indexer-update.sh` (and the indexers not re-provisioned).  (I probably should have explicitly made it clear that we don't install yq automatically yet.)

Reply:
Yeah, I filed another bug for that https://bugzilla.mozilla.org/show_bug.cgi?id=1672014 and it has a separate PR.